index.js

```diff
@@ -46,28 +46,32 @@ module.exports = {
     if (type !== 'all' || this.app.options.__is_building_fastboot__) {
       return tree;
     }
+    const trees = [tree];
+    const destDirIsRoot = this.options.destDir === '';
 
-    const staticBootTree = new Funnel(new StaticBootBuild(tree, {
+    trees.push(new Funnel(new StaticBootBuild(tree, {
       paths: this.options.paths
     }), {
       include: this.options.include,
       srcDir: './',
       destDir: this.options.destDir
-    });
+    }));
 
-    const assetsTree = new Funnel(tree, {
-      include: ['**/*'],
-      srcDir: 'assets',
-      destDir: 'staticboot/assets'
-    });
+    if (!destDirIsRoot) {
+      trees.push(new Funnel(tree, {
+          exclude: ['fastboot/**/*', 'index.html', 'tests/**/*', 'testem.js'],
```

Comment:
Noticed that a `package.json` file keeps appearing the in the `destDir`, not sure if you want to add that to the list of excluded files.